bin/run-l10n-extraction

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+
+# Exit immediately when a command fails.
+set -e
+# Make sure exit code are respected in a pipeline.
+set -o pipefail
+# Treat unset variables as an error an exit immediately.
+set -u
+
+INITIAL_GIT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+DIFF_WITH_ONE_LINE_CHANGE="1 file changed, 1 insertion(+), 1 deletion(-)"
+
+info() {
+  local message="$1"
+
+  echo ""
+  echo "INFO: $message"
+  echo ""
+}
+
+error() {
+  local message="$1"
+
+  echo "ERROR: $message"
+  exit 1
+}
+
+run_l10n_extraction() {
+  local app="$1"
+  local branch="$app-locales"
+
+  if [[ "$INITIAL_GIT_BRANCH" != "master" ]]; then
+    error "Please make sure you are on the 'master' branch."
+  fi
+
+  # Make sure the 'master' branch is up-to-date.
+  git pull origin master
+
+  # Ensure the branch to extract the locales is clean.
+  if [[ $(git branch --list "$branch") ]]; then
+    info "Deleting branch '$branch' because it already exists"
+    git branch -D "$branch"
```

Comment:
should we delete the remote branch too?

Reply:
No, because we need it to open the pull request. That being said, we might want to force push the remote branches in case we didn't delete it previously (github should automatically delete a git branch when the corresponding PR is merged).